db/migrate/20150915001329_assign_tenant_to_miq_request.rb

```diff
@@ -0,0 +1,44 @@
+class AssignTenantToMiqRequest < ActiveRecord::Migration
+  class Tenant < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+
+    # seed and return the current root_tenant
+    def self.root_tenant
+      Tenant.where(:ancestry => nil).first || Tenant.create!(:use_config_for_attributes => true)
+    end
+  end
+
+  class MiqGroup < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  class MiqRequest < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  class MiqRequestTask < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  class Service < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  class ServiceTemplate < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def change
+    models = [MiqRequest, MiqRequestTask, Service, ServiceTemplate]
```

Comment:
Nix MiqGroup... I think we still need: ServiceTemplateCatalog, Provider, MiqAeNamespace, ExtManagementSystem